client/state/stored-cards/assembler.js

```diff
@@ -0,0 +1,11 @@
+export const createStoredCard = card => {
```

Comment:
We should keep it small here, too. Let's leave it for the future :D
BTW, when I thought a bit more about mapping in the selectors, I figure out it can have some performance implications. Probably each time you cal selector you are going to return new object/array, so that can get tricky if you don't use utils like `reselect` :)